### ISSUE TITLE
Crack WPA Error - Password is on dic file but Error is -> Passphrase not in dictionary

### ISSUE THREAD

**Issue by thesagebr** [derv82/wifite2#234]
I have this situation here, I setup the Wifi (sharing on mobile) and one on another router, two networks with one password that is on rockyou.txt, but I got the same error on bolth: Passphrase not in dictionary

- **Wifite version: 2.2.5**

[+] Enter the cracking tool to use (john, hashcat, cowpatty, aircrack): hashcat

[+] Cracking 4-Way Handshake TEST-NET (3C:A1:0D:81:D3:25)
[+] Running: hcxpcaptool -o /tmp/wifitejyP3v2/generated.hccapx hs/handshake_TEST-NET_3C-A1-0D-81-D3-25_2019-08-26T16-34-39.cap
[+] Running: hashcat --quiet -m 2500 /tmp/wifitejyP3v2/generated.hccapx ./rockyou.txt --force
[+] Running: hashcat --quiet -m 2500 /tmp/wifitejyP3v2/generated.hccapx ./rockyou.txt --force --show
[!] Failed to crack TEST-NET (3C:A1:0D:81:D3:25): Passphrase not in dictionary

**Comment by prizrak-andy** [comment]
have you ever tried other tools, like johntheripper or aircrack ?

**Comment by thesagebr** [comment]
Yes, I try all on this wifite version, I don't know why but last year I have used one old version of wifite2 and works perfect.

**Comment by RaduNico** [comment]
I've described a similar issue here #174 . Could you run the commands outside wifite and check if hashcat fails with an error?
